pkg/doc/vc/profile/profile.go

```diff
@@ -40,6 +40,7 @@ type DataProfile struct {
 	Creator                 string                             `json:"creator"`
 	Created                 *time.Time                         `json:"created"`
 	DIDPrivateKey           string                             `json:"didPrivateKey"`
+	DisableVCStatus         bool                               `json:"disableVCStatus"`
```

Comment:
unrelated to this PR, but we should align on underscore JSON keys (like Sidetree).
https://github.com/trustbloc/edge-service/issues/254